examples/00_additive_single_bead.py

```diff
@@ -119,10 +111,11 @@
 # Plot melt pool statistics
 # -------------------------
 # Obtain a :class:`Pandas DataFrame <pandas.DataFrame>` containing the melt pool
-# statistics by using the :meth:`data_frame() <ansys.additive.core.single_bead.MeltPool.data_frame>`
-# property of the ``melt_pool`` attribute of the ``summary`` object. Use the
-# :meth:`DataFrame.plot() <pandas.DataFrame.plot>` method to plot the melt
-# pool dimensions as a function of bead length.
+# statistics by using the :meth:`~MeltPool.data_frame` method of the ``melt_pool``
+# attribute of the ``summary`` object. The column names for the :class:`~pandas.DataFrame`
+# are described in the documentation for :meth:`~MeltPool.data_frame`. Use the
```

Comment:
```suggestion
# instance are described in the documentation for :meth:`~MeltPool.data_frame`. Use the
```
A code entity should have a noun describing what type of object it is. I'm guessing function in the few comments I've made. However, please revise if some other noun should be used.